src/store.ts

```diff
@@ -168,6 +173,8 @@ export interface IStore {
 
     getDeviceId?(): string;
     setDeviceId?(deviceId: string): void;
+    getUserAttributes?(mpid: MPID): UserAttributes;
+    setUserAttributes?(mpid: MPID, attributes: UserAttributes): void;
```

Comment:
There is both an API for `setUserAttribute`, as well as `setUserAttributes`.  The latter is included here, but not the former.  Should the former also be included here?

Reply:
`setUserAttribute` and `setUserAttributes` are part of the Identity API, but internally, they all write to persistence the same (ish) way. Fort the Store, we only store user attributes as `ua`, so we probably only need a single setter/getter.